db/seeds.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+Member.create(
+  username:  'suee',
```

Comment:
For a number of reasons, I would recommend not using real people or contact details here. Consider creating one or two fixed fake members you can use to login with for development purposes _(e.g. `member1@member-portal.dev`/`test1234`)_, and then use something like [`faker`](https://github.com/stympy/faker) to dynamically populate fake users for testing things like pagination, search, etc. as needed.